include/open62541/server.h

```diff
@@ -1909,6 +1909,45 @@ UA_Server_createSigningRequest(UA_Server *server,
                                const UA_ByteString *nonce,
                                UA_ByteString *csr);
 
+/* Adds certificates and Certificate Revocation Lists (CRLs) to a specific certificate group
+ * on the server.
+ *
+ * @param server The server object
+ * @param certificateGroupId The NodeId of the certificate group where certificates will be added
+ * @param certificates The certificates to be added
+ * @param certificatesSize The number of certificates
+ * @param crls The associated CRLs for the certificates, required when adding issuer certificates
+ * @param crlsSize The number of CRLs
+ * @param isTrustedCertificates Indicates whether the certificates should be added to the trusted list or the issuer list
+ * @param appendCertificates Indicates whether the certificates should be added to the list or replace the existing list
+ * @return ``UA_STATUSCODE_GOOD`` on success
+ */
+UA_StatusCode UA_EXPORT
+UA_Server_addCertificates(UA_Server *server,
+                          const UA_NodeId certificateGroupId,
+                          UA_ByteString *certificates,
+                          size_t certificatesSize,
+                          UA_ByteString *crls,
+                          size_t crlsSize,
+                          const UA_Boolean isTrustedCertificates,
+                          const UA_Boolean appendCertificates);
```

Comment:
Should "isTrustedCertificates" be renamed to "areTrustedCertificates"?

And also just a note, not a change request:
The name "addCertificates" hints that certificates are added = appended to some collection. Still there is a flag to choose between appending and replacing. My initial thought was to rename the function to "setCertificates" but this feels even worse.

Reply:
That could be simply "isTrusted".